envs/echo.yml

```diff
@@ -0,0 +1,156 @@
+  - libclang=10.0.1=default_hb85057a_2
+  - libcurl=7.88.1=h91b91d3_2
+  - libdeflate=1.17=h5eee18b_1
+  - libedit=3.1.20191231=he28a2e2_2
+  - libev=4.33=h7f8727e_1
+  - libevent=2.1.12=h8f2d780_0
+  - libffi=3.4.4=h6a678d5_0
+  - libgcc=7.2.0=h69d50b8_2
+  - libgcc-ng=13.2.0=h77fa898_7
+  - libllvm10=10.0.1=hbcb73fb_5
+  - libnghttp2=1.52.0=ha637b67_1
+  - libpng=1.6.39=h5eee18b_0
+  - libpq=12.15=h37d81fd_1
+  - libssh2=1.10.0=h37d81fd_2
+  - libstdcxx-ng=13.2.0=hc0a3c3a_7
+  - libtiff=4.5.1=h6a678d5_0
+  - libuuid=1.41.5=h5eee18b_0
+  - libwebp=1.3.2=h11a3e52_0
+  - libwebp-base=1.3.2=h5eee18b_0
+  - libxcb=1.15=h7f8727e_0
+  - libxkbcommon=1.0.1=hfa300c1_0
+  - libxml2=2.9.14=h74e7548_0
+  - llvm-openmp=14.0.6=h9e868ea_0
+  - lz4-c=1.9.4=h6a678d5_0
+  - matplotlib=3.7.2=py38h06a4308_0
+  - matplotlib-base=3.7.2=py38h1128e8f_0
+  - mkl=2023.1.0=h213fc3f_46344
+  - mkl-service=2.4.0=py38h5eee18b_1
+  - mkl_fft=1.3.8=py38h5eee18b_0
+  - mkl_random=1.2.4=py38hdb19cb5_0
+  - monotonic=1.5=py_0
+  - msgpack-python=1.0.3=py38hd09550d_0
+  - munkres=1.0.7=py_1
+  - ncurses=6.4=h6a678d5_0
+  - nspr=4.35=h6a678d5_0
+  - nss=3.89.1=h6a678d5_0
+  - numexpr=2.8.4=py38hc78ab66_1
+  - numpy=1.24.3=py38hf6e8229_1
+  - numpy-base=1.24.3=py38h060ed82_1
+  - openjpeg=2.4.0=h3ad879b_0
+  - openpyxl=3.0.10=py38h5eee18b_0
+  - openssl=1.1.1w=h7f8727e_0
+  - packaging=23.1=py38h06a4308_0
+  - pandas=2.0.3=py38h1128e8f_0
+  - pcre=8.45=h295c915_0
+  - pillow=10.0.1=py38ha6cbd5a_0
+  - pip=23.3=py38h06a4308_0
+  - pkgutil-resolve-name=1.3.10=py38h06a4308_0
+  - ply=3.11=py38_0
+  - pycparser=2.21=pyhd3eb1b0_0
+  - pyopenssl=23.2.0=py38h06a4308_0
+  - pyparsing=3.0.9=py38h06a4308_0
+  - pyqt=5.15.10=py38h6a678d5_0
+  - pyqt5-sip=12.13.0=py38h5eee18b_0
+  - pysocks=1.7.1=py38h06a4308_0
+  - python=3.8.18=h7a1cb2a_0
+  - python-dateutil=2.8.2=pyhd3eb1b0_0
+  - python-tzdata=2023.3=pyhd3eb1b0_0
+  - pytz=2023.3.post1=py38h06a4308_0
+  - pyyaml=6.0.1=py38h5eee18b_0
+  - qt-main=5.15.2=h327a75a_7
+  - readline=8.2=h5eee18b_0
+  - referencing=0.30.2=py38h06a4308_0
+  - rpds-py=0.10.6=py38hb02cf49_0
+  - samtools=1.5=1
+  - setuptools=68.0.0=py38h06a4308_0
+  - simplejson=3.17.6=py38h7f8727e_0
+  - singularity=2.4.2=0
+  - sip=6.7.12=py38h6a678d5_0
+  - six=1.16.0=pyhd3eb1b0_1
+  - sqlite=3.41.2=h5eee18b_0
+  - swagger-spec-validator=3.0.3=pyhd8ed1ab_0
+  - tbb=2021.8.0=hdb19cb5_0
+  - tk=8.6.12=h1ccaba5_0
+  - toml=0.10.2=pyhd3eb1b0_0
+  - tomli=2.0.1=py38h06a4308_0
+  - tornado=6.3.3=py38h5eee18b_0
+  - typing=3.10.0.0=py38h06a4308_0
+  - typing-extensions=4.7.1=py38h06a4308_0
+  - typing_extensions=4.7.1=py38h06a4308_0
+  - wheel=0.37.1=pyhd3eb1b0_0
+  - xlsx2csv=0.8.1=pyhd8ed1ab_0
+  - xmltodict=0.12.0=pyhd3eb1b0_0
+  - xz=5.4.2=h5eee18b_0
+  - yaml=0.2.5=h7b6447c_0
+  - yq=2.12.0=pyhd3eb1b0_0
+  - zipp=3.11.0=py38h06a4308_0
+  - zlib=1.2.13=h5eee18b_0
+  - zstd=1.5.5=hc292b87_0
+  - pip:
+    - charset-normalizer==3.3.2
+    - facetsapi==0.5.0
```

Comment:
We can remove the facetsapi from env file and instead add a line in the Readme to run the facetsapi install step after the conda env setup.